Cargo.toml

```diff
@@ -1,3 +1,3 @@
 [workspace]
 members = ["rosidl_runtime_rs"]
-resolver = "3"
+resolver = "2"
```

Comment:
Hey sorry for the intrusion. I was taking a look at ros2_rust's examples and following the instructions for setting up the workspace led me to this error (resolver 3 not available for 1.75).
Going in with this PR would be great so as to build the workspace from scratch without issues, especially for newcomers.